release_scripts/signassert.py

```diff
@@ -0,0 +1,63 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2011-2015, Armory Technologies, Inc.                           #
+# Distributed under the GNU Affero General Public License (AGPL v3)            #
+# See LICENSE or http://www.gnu.org/licenses/agpl.html                         #
+#                                                                              #
+################################################################################
+
+import getpass
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+
+from armoryengine.ALL import *
+from jasvet import ASv1CS, readSigBlock, verifySignature
```

Comment:
IIRC, we're trying to get away from jasvet. This could be a problem down the road. For now, we'll go with it. The alternative (find and add new code) isn't better by any means.